src/Inventory.java

```diff
@@ -1,32 +1,66 @@
+/**
+* Professor: 	Orlando Montalvo
+* Class: 		Object Oriented Analysis & Design CSC7400
+* @author 		Mohamed Fadili
+* @version 		1.0
+* @since   		09-11-2017
+*/
+
 import java.util.Iterator;
 import java.util.LinkedList;
 import java.util.List;
 
 public class Inventory {
-  private List guitars;
+  private List<Guitar> guitars;
 
-  public Inventory() {
-    guitars = new LinkedList();
+  /**
+ * this class will create a Guitar search a get a Guitar if any from 
+ */
+public Inventory() {
+    guitars = new LinkedList<Guitar>();
   }
 
-  public void addGuitar(String serialNumber, double price,
+  /**
+   * create new Guitar using the parameters below
+   * 
+ * @param serialNumber
+ * @param price
+ * @param builder
+ * @param model
+ * @param type
+ * @param backWood
+ * @param topWood
+ */
+public void addGuitar(String serialNumber, double price,
                         String builder, String model,
                         String type, String backWood, String topWood) {
     Guitar guitar = new Guitar(serialNumber, price, builder,
                                model, type, backWood, topWood);
     guitars.add(guitar);
   }
-  public Guitar getGuitar(String serialNumber) {
-    for (Iterator i = guitars.iterator(); i.hasNext(); ) {
+  /**
+   * searches for Guitar using serialNumber
+   * return Guitar if found else return null
+ * @param serialNumber
+ * @return guitar 
```

Comment:
Ok. Not bad.  Let's reorder a little.
/**
 * searches for Guitar using serialNumber
 * @param serialNumber guitar serial number
 * @return guitar if found else return null